### PR TITLE
Orgs / User Route Refactor

### PR DESCRIPTION
**Pull request by sebastinez**
This PR refactors the `/users/:user` and `/orgs/:orgs` routes and replaces them with `/:addressOrName` routes.
Also the projects in org or user context are simplified to `/:addressOrName/:projects`.

Big points:
- Extract all the anchored project querying into `base/orgs/View/Projects.svelte`
- Join the `orgs/View` and `/users/View` components into `Profile.svelte`

### CODE REVIEW

File: src/base/orgs/View/Projects.svelte [hunk 1]
```diff
@@ -40,36 +43,65 @@
...
-          </Widget>
-        {/if}
-      </div>
-    {/each}
-  {:catch err}
-    <Message error>
-      <strong>Error: </strong> failed to load projects: {err.message}.
-    </Message>
-  {/await}
+  {#if profile.org}
```
Comment (sebastinez) [inline src/base/orgs/View/Projects.svelte:46]: Okay this is to fetch all the org projects and else query the `profile.anchorsAccount` projects..
We could also separate them into separate components.. eg. `PendingProjects` `AnchoredProjects`

File: src/profile.ts [hunk 2]
```diff
@@ -128,31 +141,52 @@ export class Profile {
...
+    let type = AddressType.EOA;
+    let org: Org | null = null;
     const ens = await resolveEnsProfile(addressOrName, profileType, config);
 
     if (ens) {
       if (ens.address) {
+        type = await identifyAddress(ens.address, config);
+
+        if (type === AddressType.Org) {
+          org = await Org.get(ens.address, config);
```
Comment (sebastinez) [inline src/profile.ts:153]: The nicest way I found for Orgs is to be part of the Profile type as a kind of subset, to allow us to query for orgs and get all the properties and methods linked to.

File: src/base/projects/Widget.svelte [hunk 3]
```diff
@@ -35,13 +34,13 @@
     }
   });
 
+  $: console.log(addressOrName);
```
Comment (cloudhead) [inline src/base/projects/Widget.svelte:37]:
```suggestion
```

File: src/Profile.svelte [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+  };
+  $: isOrgAuthorized = async (org: Org): Promise<boolean> => {
+    if ($session) {
+      if (isOwner(org)) {
+        return true;
+      }
+      return await org.isMember($session.address, config);
+    }
+    return false;
+  };
```
Comment (cloudhead) [inline src/Profile.svelte:69]: I know this wasn't introduced in this PR, but why is it that all these functions are defined as reactive?

File: src/Profile.svelte [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+  let setNameForm: typeof SvelteComponent | null =
+    action === "setName" ? SetName : null;
+  const setName = () => {
+    setNameForm = SetName;
+  };
+
+  let transferOwnerForm: typeof SvelteComponent | null = null;
+  const transferOwnership = () => {
+    transferOwnerForm = TransferOwnership;
+  };
```
Comment (cloudhead) [inline src/Profile.svelte:34]:
```suggestion
  };
  
```

File: src/Profile.svelte (hunk 4 above)
Comment (sebastinez) [reply src/Profile.svelte:74]: They are required to change the permissions on setting profile and changing owner, eg when the logged in user changes.

File: src/base/orgs/Routes.svelte [hunk 6]
```diff
@@ -12,6 +11,6 @@
   <Index {config} />
 </Route>
 
-<Route path="/orgs/:address" let:params let:location>
-  <View {config} addressOrName={params.address} action={getSearchParam("action", location)} />
```
Comment (sebastinez) [inline src/base/orgs/Routes.svelte:16]: I removed the action param here, since this is only a redirection.

File: src/Profile.svelte (hunk 4 above)
Comment (cloudhead) [reply src/Profile.svelte:74]: Hmm not sure that answers my question, since they are functions that would have to be called anyway.. I'll look into it.

**Review by cloudhead — APPROVED** [review]

**Pull request merged**